### PR TITLE
fix portfolio default value

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
None default value creates exceptions on real trading without being authenticated

### CODE REVIEW

File: octobot_trading/data/portfolio.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def __init__(self, exchange_name, is_simulated=False):
 
         self.logger = get_logger(f"{self.__class__.__name__}{'Simulator' if is_simulated else ''}[{exchange_name}]")
         self.lock = Lock()
-        self.portfolio = None
+        self.portfolio = {}
```
Comment (Herklos) [inline octobot_trading/data/portfolio.py:43]: ?

Comment (Herklos) [reply octobot_trading/data/portfolio.py:43]: A portfolio should be initialize and will be set to `{}` during initialize()

Comment (GuillaumeDSM) [reply octobot_trading/data/portfolio.py:43]: Using None is producing a lot of exceptions at different places when portfolio is never initialized (when exchange creds are wrong / missing for ex)

Comment (GuillaumeDSM) [reply octobot_trading/data/portfolio.py:43]: I don't thing having a None default value is adding any value to this

Comment (Herklos) [reply octobot_trading/data/portfolio.py:43]: It doesn't make sense to create a portfolio and not initialize it.

**Pull request merged**